satellite/satellitedb/migrate.go

```diff
@@ -430,6 +430,47 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						rollup_end_time )`,
 				},
 			},
+			{
+				Description: "Merge overlay_cache_nodes into nodes table",
+				Version:     10,
+				Action: migrate.SQL{
+					// Add the new columns to the nodes table
+					`ALTER TABLE nodes ADD address TEXT;
```

Comment:
you can try `ALTER TABLE nodes ADD address TEXT NOT NULL DEFAULT ''` (my sql might not 100% correct) and later only drop DEFAULT constraint if necessary `ALTER TABLE nodes ALTER COLUMN address DROP DEFAULT`

Reply:
Nice hint. Thanks!